winit-core/src/window.rs

```diff
@@ -850,12 +870,15 @@ pub trait Window: AsAny + Send + Sync + fmt::Debug {
     /// - **iOS / Android / Web / Orbital:** Unsupported.
     fn set_surface_resize_increments(&self, increments: Option<Size>);
 
-    /// Modifies the title of the window.
+    /// The window transparency state.
+    ///
+    /// Getter for [`set_transparent`][Window::set_transparent], see that for details.
     ///
     /// ## Platform-specific
     ///
-    /// - **iOS / Android:** Unsupported.
-    fn set_title(&self, title: &str);
+    /// - **iOS / Android / Web:** Unsupported.
```

Comment:
```suggestion
    /// - **iOS / Android / Web:** Unsupported. Always returns `false`.
```
I propose if it returns a `bool`, we should specify what it returns in case its unsupported or unimplemented.